### ISSUE TITLE
[TabsContainer] - Second tab disappears when changing orientation and back

### ISSUE THREAD

**Issue by kaYcee** [mlaursen/react-md#782]
### Description
I am using a minimal TabsContainer component with just two empty tabs, no styles, no adjustments. When the second tab is selected in portrait mode of my mobile phones (iPhone 8, HTC530), then orientation switched to landscape by rotating the phone, and then rotated back into portrait mode, the second tab is still marked selected but the title disappeared. What is more, when I select the first tab again the second tab cannot be selected anymore. Only re-orienting the phone lets the tab title appear and make selectable again.

Interestingly, I could reproduce the issue in a minimal example on jsfiddle (with a very narrow display area) but failed when using the codepen template, which uses much more space for the result window. The issue seems to occur primarily on small viewports and seems to be related to the constant `translate3d`-operations being performed when re-oriented.

P.S. Also tried with 3 tabs. Then the issue only appears when the second tab is selected.

### Link to a gist or code sample where the issue can be reproduced

https://jsfiddle.net/9eazdb2r/4/ (issue reproducible every time)
https://codepen.io/anon/pen/VGLaQV (issue not reproducible)
### Version
- React - 16.4.2
- React-MD - 1.5
- Browser - both Safari on iOS 11.4.1 on iPhone and Chrome 66 on HTC530.

**Comment by kaYcee** [comment]
Hi @mlaursen! I think I have found the source of this bug. in `Tabs/Tabs.js, Line 307` it is checked whether the `mobile` property is set or not. When it is set, the `overflowAtIndex`-variable is computed. With the second tab active and the phone first rotated and then re-rotated, the resize event seems to be called numerous times, causing this bug. As a temporary solution I've applied `mobile={true}` to all my usages and this seems to circumvent the issue.
